doc/rst/language/spec/conversions.rst

```diff
@@ -575,14 +573,12 @@ type. Such a conversion does not change the value of the expression.
 Explicit Numeric Conversions
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Explicit conversions are allowed from any numeric type or boolean to
+Explicit conversions are allowed from any numeric type or ``bool`` to
 bytes or string, and vice-versa.
```

Comment:
I suggest being consistent and either back-tick "bytes" and "string" or un-back-tick "bool". While there, let us explain what these casts do. For example:

A conversion to bytes or string produces the string representation of the number or the strings "true" or "false" for the boolean. The opposite conversion produces the numeric or boolean value represented by the string or bytes or throws an IllegalArgumentError when the string does not represent a value of the target type.